### ISSUE TITLE
Hardcoding number of cores

### ISSUE THREAD

**Issue by seabbs** [pearsonca/hbmPRNG#11]
The vignette currently hard codes 5 cores. This should really be clearly communicated to the user and be set at the beginning of the vignette.

**Comment by pearsonca** [comment]
Yeah, I've fixed this in the inst version, which the vignette will just load. I'll propagate the code changes to the vignette as well.
